src/nvim/ex_docmd.c

```diff
@@ -76,6 +76,17 @@
 #include "nvim/event/rstream.h"
 #include "nvim/event/wstream.h"
 
+/// The scope of a command.
+///
+/// The higher a number, the deeper a scope.
+enum cd_scope {
+  kCdScopeEditor, ///< Affects the entire instance of NeoVim.
+  kCdScopeTab,    ///< Affects on tab page.
+  kCdScopeWindow, ///< Affects one window.
+};
+
+#define MAX_CD_SCOPE  kCdScopeWindow  ///< Total number of scoped defined.
```

Comment:
`///<` (with `<`) comments are for class/struct/enum/union/… members. Macros is not a member.